counterparty-core/counterpartycore/lib/api/api_v1.py

```diff
@@ -563,12 +564,47 @@ def generate_create_method(tx):
             def create_method(**kwargs):
                 try:
```

Comment:
Does the code between here and the db transaction potentially throw an exception? If not we could move the try lower.

Reply:
fixed